pysmt/cmd/installers/msat.py

```diff
@@ -103,10 +103,10 @@ def move(self):
             if f.endswith(".so") or f.endswith(".dll") or f.endswith(".dylib"):
                 SolverInstaller.mv(os.path.join(libdir, f), self.bindings_dir)
 
-        # Fix issue in MathSAT 5.5.1 linking to incorrect directory on OSX
+        # Fix issue in MathSAT 5.6.10 linking to incorrect directory on OSX
```

Comment:
Minor. If this is still happening, I wonder if we can try to help the team fix their building process.

Reply:
I think so, but I have not checked